js/views/ConnectedPeersPage.js

```diff
@@ -11,6 +12,48 @@ export default class extends baseVw {
     }
 
     this.peers = options.peers;
+    this.peersLoaded = 0;
+    this.peersIterator = 12;
+  }
+
+  className() {
+    return 'userPage';
+  }
+
+  events() {
+    return {
+      'click .js-morePeersBtn': 'loadPeers',
+    };
+  }
+
+  get $peerWrapper() {
+    return this._$peerWrapper ||
+        (this._$peerWrapper = this.$('.js-peerWrapper'));
+  }
+
+  get $morePeers() {
+    return this._$morePeers ||
+        (this._$morePeers = this.$('.js-morePeers'))
+  }
+
+  loadPeers() {
+    if (this.peers.length > this.peersLoaded) {
+      this.peers.slice(this.peersLoaded, this.peersLoaded + this.peersIterator).forEach((peer) => {
+        console.log(peer)
+        const user = this.createChild(userShort, {
+          guid: peer,
+        });
+        this.$peerWrapper.append(user.render().$el);
```

Comment:
Could we add these to a document fragment and then add the whole fragment instead of one by one to the DOM?